src/scratch/asteroids/README.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Lærerveiledning - Asteroids
+level: 4
+language: nb
+---
+
+# Om oppgaven {.activity}
+På slutten av 1970-tallet ga Atari ut to spill hvor man skulle
+kontrollere et romskip. Det første var Lunar Lander, men dette ble
+utkonkurrert av Asteroids som Atari ga ut noen måneder senere.
+Spillene var faktisk så like at de kunne gjenbruke mye av teknologien.
+Vi skal gjøre det samme! Du må derfor ha laget
+[Lunar Lander](../lunar_lander/lunar_lander.html) før du begynner på
+dette prosjektet. I Asteroids er målet å beskytte romskipet mot
+asteroider ved å skyte dem i småbiter.
+
+![](asteroids.png)
+
+## Oppgaven passer til: {.check}
+ __Fag__: Kunst og håndtverk, matematikk, programmering.
+__Anbefalte trinn__: 3.-10. trinn.
```

Comment:
Når den dekker kompetansemål for 3.-4. trinn har eg lagt inn at den passar frå 3. trinn. Men det kan godt vere slik at me opplyser om at den dekker kompetansemål me ikkje anbefaler at oppgåva blir gitt for å dekke.